apps/guest/src/home/components/const/qualify.const.ts

```diff
@@ -0,0 +1 @@
+export const TEXT_RATE = ['veryBad', 'bad', 'fair', 'good', 'excellent'];
```

Comment:
```suggestion
export const TEXT_RATE = ['modalQualifyComponent.veryBad', 'modalQualifyComponent.bad', 'modalQualifyComponent.fair', 'modalQualifyComponent.good', 'modalQualifyComponent.excellent'];
```

I think it is better becouse when you will use it, you don`t have to depend on prefix  `modalQualifyComponent`